### PR TITLE
Pass all dataframes in serialized format

### PR DESCRIPTION
**Pull request by jdebacker**
This PR updates `btax.front_end_util.runner_json_tables()` to pass all data frames in a serialized format.

### CODE REVIEW

File: btax/front_end_util.py [hunk 1]
```diff
@@ -44,7 +54,10 @@ def runner_json_tables(test_run=False,start_year=2016,iit_reform=None,**user_par
                     tables[k1][k2] = v2
         else:
             raise ValueError('Expected an "asset" or "industry" related table')
-    return add_summary_rows_and_breaklines(dict(tables), start_year, do_assertions=test_run)
+    return json.dumps({
+        'json_table': add_summary_rows_and_breaklines(dict(tables), start_year, do_assertions=test_run),
+        'dataframes': serialized_dataframes,
+        })
```
Comment (hdoupe) [inline btax/front_end_util.py]: @jdebacker I think this should be: 
```
+    return {
+        'json_table': add_summary_rows_and_breaklines(dict(tables), start_year, do_assertions=test_run),
+        'dataframes': serialized_dataframes,
+        }) 
```
Before, a Python dictionary was passed back to webapp.  The new code returns a string but the webapp expects a dictionary.  Making this change resolves one of the webapp compatibility errors.

Comment (jdebacker) [inline btax/front_end_util.py]: @hdoupe If that works with the webapp, I'm glad to merge a PR that makes the change you propose.  When choosing the route I did, I wasn't sure if the webapp needed everything to be serialized so used the `json.dump()`

Comment (hdoupe) [inline btax/front_end_util.py]: @jdebacker It works with the webapp.  Thanks for making the change.

Comment (jdebacker) [inline btax/front_end_util.py]: @hdoupe Want to clarify - do I need to make the change your proposed or is what was merged with this PR ok?

Comment (hdoupe) [inline btax/front_end_util.py]: @jdebacker Yes, you need to make the change.

**Pull request merged**